### ISSUE TITLE
Switching Quiver and a Bag Produces Artifacts

### ISSUE THREAD

**Issue by nonameform** [shagu/pfUI#1364]
Initial bag order.

![WoWScrnShot_110924_213625](https://github.com/user-attachments/assets/ca3fd6f7-77c1-40ee-9a3f-55290ed2c598)

Two bags are switched. Everything is fine.

![WoWScrnShot_110924_213631](https://github.com/user-attachments/assets/fd4a2f4e-263f-40be-a10b-c6637d5988d7)

Then I switch bags back and switch one bag and quiver. There is an artifact in the lower right corner and the quiver is visually split in two parts.

![WoWScrnShot_110924_213642](https://github.com/user-attachments/assets/9987826a-e51f-4c5f-9bb7-fd25bf007d84)

Bag and quiver are switched back, but the quiver is still 'split'.

![WoWScrnShot_110924_213707](https://github.com/user-attachments/assets/88071e28-39b6-4326-b9a5-8cace61d9eb4)

After reloading UI.

![WoWScrnShot_110924_213726](https://github.com/user-attachments/assets/9d7b61e0-357e-4d50-8e00-89f95554ebde)

**Comment by shagu** [comment]
Sorry, i'm unable to reproduce it:
[Screencast From 2024-11-04 10-55-41.webm](https://github.com/user-attachments/assets/63350b75-10fe-4fea-a7c2-8a57a0d9427d)

**Comment by nonameform** [comment]
I wonder if there might be some leftover settings from previous pfUI installations that produce that visual bug? I'm using the same pfUI.lua file from SavedVariables that I had for several years now just to copy settings between servers that run from different folders on my PC.

That happens both on Kronos and on Turtle WoW for me.

**Comment by nonameform** [comment]
I removed my settings completely and ran pfUI as if for the first time. It was the only addon running and still that bug occurred.

Also I discovered that if I remove a bag, the icon will stay in the bag bar as if the bag is still equipped, though the tooltip will be correct. On this screenshot second slot is empty, the bag is unequipped, yet the icon remains.

![WoWScrnShot_111224_145205](https://github.com/user-attachments/assets/2f3f9224-2779-4a08-8dcb-df9bfde93318)

Another discovery I made is that by removing a bag and than equipping it fixes the bug I initially reported.

And the last discovery I made: it happens not only with quiver. I switched bags 1 and 3 (counting from the left). The tooltip is for the big icon and says "Roasted Quail" yet shows HS with CD.

![WoWScrnShot_111224_143205](https://github.com/user-attachments/assets/2dfa73ee-d716-44d2-a86f-fb6b80f42b51)

That is weird as it happens on two different servers with two different installations. I even checked on a different PC. and the bug happens there as well.
